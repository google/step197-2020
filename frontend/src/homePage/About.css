.About{
    background-color:#C4C4C4;
    height:10vh;
}