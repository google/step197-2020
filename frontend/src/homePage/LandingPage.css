.landingPage {
    height:60vh;
    display: flex;
    flex-direction: column;
    align-items: center;
    justify-content: center;
}

.landingPage h1 {
    font-size: 64px;;
}