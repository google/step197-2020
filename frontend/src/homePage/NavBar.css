/* Add a black background color to the top navigation */
.navBar {
    background-color: #5A90A1;
    overflow: hidden;
    display:flex;
    justify-content:flex-end;
    padding:10px 0;
    position: fixed;
    top:0;
    width:100%;
  }