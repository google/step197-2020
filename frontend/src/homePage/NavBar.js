import React, {Component} from "react";
import css from "./NavBar.css";
import LoginButton from "./LoginButton.js";
import {motion} from "framer-motion"

class NavBar extends React.Component {
  constructor(props) {
    super(props);
    this.html = null;
    this.message = "Login";
    // Check if the user is logged in to display folders
    if (this.props.loginStatus) {
      this.html = (
        <LoginButton
          status="My Folders"
          color="#F4B400"
          fontSize="24px"
          clickFunction={this.props.folderClick}
        ></LoginButton>
      );
      this.message = "Logout";
    }
  }

  render() {
    return (
      <div className="navBar">
        <motion.div whileHover={{ scale: 1.1 }}>{this.html}</motion.div>
        <motion.div whileHover={{ scale: 1.1 }}>
          <LoginButton
            status={this.message}
            clickFunction={this.props.loginClick}
            color="#C4C4C4"
            fontSize="24px"
          ></LoginButton>
        </motion.div>
      </div>
    );
  }
}

export default NavBar;