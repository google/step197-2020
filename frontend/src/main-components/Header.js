import React from "react";
import styled from "@emotion/styled";
import Search from "../sub-components/Search";
import logo from "../SVG/Logo.jpg";
import "material-design-icons";

const Header = (props) => {
  
  const Head = styled.div`
    display: flex;
    align-items: center;
    border: 1rem solid white;
    border-radius: 1rem;
    background-color: white;
    margin 1%;
    @media (max-width: 700px) {
        text-align: left;
    }
  `;
  const Image = styled.img`
    width: 70%;
    padding-right: 15%;
    padding-left: 15%;
  `;
  const Box = styled.div`
    flex: 2;
    @media (max-width: 700px) {
      flex: 1;
      margin-right: 0.5rem;
    }
  `;
  const Menu = styled.button`
    background-color: white;
    height: 100%;
    padding-right: none;
    border: none;
  `;

  let display = <Image src={logo} alt="Frame Cards Logo"></Image>;
  /*
   * If the device size is less than 700px wide then we will display
   * a button for our side menu instead of our logo.
   */
  const width = self.innerWidth;
  if (width < 700) {
    display = (
      <Menu onClick={(e)=> props.handleClick()}>
        <i className="material-icons">menu</i>
      </Menu>
    );
  }
    return (
      <Head className="flex-container">
        <Box>{display}</Box>
        <Search></Search>
      </Head>
    );
};


export default Header;
