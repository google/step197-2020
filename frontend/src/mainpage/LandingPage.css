.LandingPage{
    height:80vh;
    display: flex;
    flex-direction: column;
    align-items: center;
    justify-content: center;
}

.LandingPage h1{
    font-size: 64px;;
}