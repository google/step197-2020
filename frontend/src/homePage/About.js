import React from 'react';
import './About.css';
import Background from "./BackgroundVideo";

class About extends React.Component {
    constructor(props) {
      super(props);
      this.state = {
      }
    }
  
    render() {
      return (
        <div className="about">
          <Background></Background>
        </div>
      );
    }
  }

export default About;