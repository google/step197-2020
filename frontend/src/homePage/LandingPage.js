import React from "react";
import "./LandingPage.css";
import LoginButton from "./LoginButton.js";
import { motion } from "framer-motion";

class LandingPage extends React.Component {
  constructor(props) {
    super(props);
    this.message = "Get Started";
    this.color = "#5A90A1";
    this.clickHandling = this.props.loginClick;
    if (this.props.loginStatus) {
      this.message = "My Folders";
      this.color = "#F4B400";
      this.clickHandling = this.props.folderClick;
    }
  }

  render() {
    return (
      <div className='landingPage'>
        <h1>Frame.Cards learning made easy</h1>
        <motion.div whileHover={{ scale: 1.2 }}>
          <LoginButton
            status={this.message}
            color={this.color}
            fontSize='32px'
            clickFunction={this.clickHandling}></LoginButton>
        </motion.div>
      </div>
    );
  }
}

export default LandingPage;