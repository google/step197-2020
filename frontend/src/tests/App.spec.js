import React from "react";
import renderer from "react-test-renderer";
import About from "../homePage/About";
import LandingPage from "../homePage/LandingPage";
import NavBar from "../homePage/NavBar";
import Header from "../main-components/Header";
import SideBar from "../main-components/Sidebar";
import CreateCard from "../pages/CreateCard";
import CreateFolder from "../pages/CreateFolder";
import MyFolders from "../pages/MyFolders";
import InsideFolder from "../pages/InsideFolder";
import NotFound from "../pages/404";
import { MemoryRouter } from "react-router-dom";

/**
 * Snapshot testing for home page
 */
describe("Home page", () => {
  test("snapshot About renders", () => {
    const component = renderer.create(<About />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });

  test("snapshot Landing Page renders", () => {
    const component = renderer.create(<LandingPage />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });

  test("snapshot Navigation Bar renders", () => {
    const component = renderer.create(<NavBar />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});

/**
 * Snapshot testing for main components
 */
describe("mainComponents", () => {
  test("snapshot header renders", () => {
    const component = renderer.create(<Header />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });

  test("snapshot SideBar renders", () => {
    const component = renderer.create(
      <MemoryRouter>
        <SideBar />
      </MemoryRouter>
    );
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});

/**
 * Snapshot testing for main App pages
 */
describe("mainApp", () => {
  test("snapshot My Folders renders", () => {
    const component = renderer.create(
      <MemoryRouter>
        <MyFolders />
      </MemoryRouter>
    );
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });

  test("snapshot CreateCard renders", () => {
    const component = renderer.create(
      <MemoryRouter>
        <CreateCard />
      </MemoryRouter>
    );
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });

  // Must define window.alert for jest
  window.alert = jest.fn();
  test("snapshot Create Folder renders", () => {
    window.alert.mockClear();
    const component = renderer.create(
      <MemoryRouter>
        <CreateFolder />
      </MemoryRouter>
    );
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });

  test("snapshot Inside Folder renders", () => {
    const component = renderer.create(
      <MemoryRouter>
        <InsideFolder />
      </MemoryRouter>
    );
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });

  test("snapshot 404 renders", () => {
    const component = renderer.create(
      <MemoryRouter>
        <NotFound />
      </MemoryRouter>
    );
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
