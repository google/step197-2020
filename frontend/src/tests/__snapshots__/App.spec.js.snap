// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home page snapshot About renders 1`] = `
<div
  className="About"
/>
`;

exports[`Home page snapshot Landing Page renders 1`] = `
<div
  className="LandingPage"
>
  <h1>
    Frame.cards learning made easy
  </h1>
  <div
    style={Object {}}
  >
    <div
      className="buttonBox"
    >
      <button
        className="MyButton"
        style={
          Object {
            "backgroundColor": "#5A90A1",
            "fontSize": "32px",
          }
        }
      >
         
        Get Started
         
      </button>
    </div>
  </div>
</div>
`;

exports[`Home page snapshot Navigation Bar renders 1`] = `
<div
  className="NavBar"
>
  <div
    style={Object {}}
  />
  <div
    style={Object {}}
  >
    <div
      className="buttonBox"
    >
      <button
        className="MyButton"
        style={
          Object {
            "backgroundColor": "#C4C4C4",
            "fontSize": "24px",
          }
        }
      >
         
        Login
         
      </button>
    </div>
  </div>
</div>
`;

exports[`mainApp snapshot 404 renders 1`] = `
<div
  className="css-gcm1jx"
>
  <div
    className="flex-container css-8jr5nd"
    id="side"
  >
    <div
      className="css-csay8j"
    >
      <a
        href="/myFolders"
        onClick={[Function]}
      >
        <div
          style={Object {}}
        >
          <img
            alt="icon option"
            className="css-fbcrhc"
            src="test-file-stub"
          />
        </div>
      </a>
    </div>
    <div
      className="css-csay8j"
    >
      <a
        href="/CreateCard"
        onClick={[Function]}
      >
        <div
          style={Object {}}
        >
          <img
            alt="icon option"
            className="css-fbcrhc"
            src="test-file-stub"
          />
        </div>
      </a>
    </div>
    <div
      className="css-csay8j"
    >
      <a
        href="/CreateFolder"
        onClick={[Function]}
      >
        <div
          style={Object {}}
        >
          <img
            alt="icon option"
            className="css-fbcrhc"
            src="test-file-stub"
          />
        </div>
      </a>
    </div>
    <div
      className="css-csay8j"
    >
      <a
        href="/ImageInterface"
        onClick={[Function]}
      >
        <div
          style={Object {}}
        >
          <img
            alt="icon option"
            className="css-fbcrhc"
            src="test-file-stub"
          />
        </div>
      </a>
    </div>
    <div
      className="css-csay8j"
    >
      <a
        href="/YoutubeInterface"
        onClick={[Function]}
      >
        <div
          style={Object {}}
        >
          <img
            alt="icon option"
            className="css-fbcrhc"
            src="test-file-stub"
          />
        </div>
      </a>
    </div>
    <div
      className="css-csay8j"
    >
      <a
        href="/myFolders"
        onClick={[Function]}
      >
        <div
          style={Object {}}
        >
          <img
            alt="icon option"
            className="css-fbcrhc"
            src="test-file-stub"
          />
        </div>
      </a>
    </div>
  </div>
  <h2>
    404 page not found
  </h2>
</div>
`;

exports[`mainApp snapshot Create Folder renders 1`] = `
<div
  className="App"
>
  <div
    className="flex-container css-1dc4ncl"
  >
    <div
      className="css-15s9vjo"
    >
      <img
        alt="Frame Cards Logo"
        className="css-7r85ak"
        src="test-file-stub"
      />
    </div>
    <div
      className="css-ku0l26"
    >
      <form
        action="/someServlet"
        className="css-1rerb9r"
      >
        <button
          className="material-icons css-9h4d91"
          type="submit"
        >
          <i
            className="material-icons"
          >
            search
          </i>
        </button>
        <input
          className="css-pux1ie"
          name="search-input"
          placeholder="Search.."
          type="text"
        />
      </form>
    </div>
  </div>
  <div
    id="main"
  >
    <div
      className="flex-container css-8jr5nd"
      id="side"
    >
      <div
        className="css-csay8j"
      >
        <a
          href="/myFolders"
          onClick={[Function]}
        >
          <div
            style={Object {}}
          >
            <img
              alt="icon option"
              className="css-fbcrhc"
              src="test-file-stub"
            />
          </div>
        </a>
      </div>
      <div
        className="css-csay8j"
      >
        <a
          href="/CreateCard"
          onClick={[Function]}
        >
          <div
            style={Object {}}
          >
            <img
              alt="icon option"
              className="css-fbcrhc"
              src="test-file-stub"
            />
          </div>
        </a>
      </div>
      <div
        className="css-csay8j"
      >
        <a
          href="/CreateFolder"
          onClick={[Function]}
        >
          <div
            style={Object {}}
          >
            <img
              alt="icon option"
              className="css-fbcrhc"
              src="test-file-stub"
            />
          </div>
        </a>
      </div>
      <div
        className="css-csay8j"
      >
        <a
          href="/ImageInterface"
          onClick={[Function]}
        >
          <div
            style={Object {}}
          >
            <img
              alt="icon option"
              className="css-fbcrhc"
              src="test-file-stub"
            />
          </div>
        </a>
      </div>
      <div
        className="css-csay8j"
      >
        <a
          href="/YoutubeInterface"
          onClick={[Function]}
        >
          <div
            style={Object {}}
          >
            <img
              alt="icon option"
              className="css-fbcrhc"
              src="test-file-stub"
            />
          </div>
        </a>
      </div>
      <div
        className="css-csay8j"
      >
        <a
          href="/myFolders"
          onClick={[Function]}
        >
          <div
            style={Object {}}
          >
            <img
              alt="icon option"
              className="css-fbcrhc"
              src="test-file-stub"
            />
          </div>
        </a>
      </div>
    </div>
    <div
      id="container"
    >
      <div
        id="innerContainer"
      >
        <div
          id="folderPreview"
        >
          <h2
            id="previewFolderName"
          >
            None
          </h2>
          <h4
            id="previewFolderLang"
          >
            English
          </h4>
        </div>
        <div
          id="formBox"
        >
          <ul>
            <form
              action="/userfolders"
              id="myForm"
              method="POST"
            >
              <li>
                <label>
                  Folder Name: 
                </label>
              </li>
              <li>
                <input
                  id="folderName"
                  name="folderName"
                  onBlur={[Function]}
                  placeholder="None"
                  required={true}
                  type="text"
                />
              </li>
              <li>
                <label>
                  Folder Language:
                </label>
              </li>
              <li>
                <div
                  id="scroll"
                >
                  <select
                    id="folderDefaultLanguage"
                    name="folderDefaultLanguage"
                    onChange={[Function]}
                    required={true}
                    value="English"
                  >
                    <option
                      value="Italian"
                    >
                      Italian
                    </option>
                    <option
                      value="Hindi"
                    >
                      Hindi
                    </option>
                    <option
                      value="Arabic"
                    >
                      Arabic
                    </option>
                    <option
                      value="Chinese"
                    >
                      Chinese
                    </option>
                    <option
                      value="English"
                    >
                      English
                    </option>
                    <option
                      value="French"
                    >
                      French
                    </option>
                    <option
                      value="Spanish"
                    >
                      Spanish
                    </option>
                    <option
                      value="German"
                    >
                      German
                    </option>
                    <option
                      value="Vietnamese"
                    >
                      Vietnamese
                    </option>
                    <option
                      value="Greek"
                    >
                      Greek
                    </option>
                  </select>
                </div>
              </li>
              <li>
                <input
                  id="submission"
                  type="submit"
                  value="Submit"
                />
              </li>
            </form>
          </ul>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`mainApp snapshot CreateCard renders 1`] = `
<div
  className="App"
>
  <div
    className="flex-container css-1dc4ncl"
  >
    <div
      className="css-15s9vjo"
    >
      <img
        alt="Frame Cards Logo"
        className="css-7r85ak"
        src="test-file-stub"
      />
    </div>
    <div
      className="css-ku0l26"
    >
      <form
        action="/someServlet"
        className="css-1rerb9r"
      >
        <button
          className="material-icons css-9h4d91"
          type="submit"
        >
          <i
            className="material-icons"
          >
            search
          </i>
        </button>
        <input
          className="css-pux1ie"
          name="search-input"
          placeholder="Search.."
          type="text"
        />
      </form>
    </div>
  </div>
  <div
    id="main"
  >
    <div
      className="flex-container css-8jr5nd"
      id="side"
    >
      <div
        className="css-csay8j"
      >
        <a
          href="/myFolders"
          onClick={[Function]}
        >
          <div
            style={Object {}}
          >
            <img
              alt="icon option"
              className="css-fbcrhc"
              src="test-file-stub"
            />
          </div>
        </a>
      </div>
      <div
        className="css-csay8j"
      >
        <a
          href="/CreateCard"
          onClick={[Function]}
        >
          <div
            style={Object {}}
          >
            <img
              alt="icon option"
              className="css-fbcrhc"
              src="test-file-stub"
            />
          </div>
        </a>
      </div>
      <div
        className="css-csay8j"
      >
        <a
          href="/CreateFolder"
          onClick={[Function]}
        >
          <div
            style={Object {}}
          >
            <img
              alt="icon option"
              className="css-fbcrhc"
              src="test-file-stub"
            />
          </div>
        </a>
      </div>
      <div
        className="css-csay8j"
      >
        <a
          href="/ImageInterface"
          onClick={[Function]}
        >
          <div
            style={Object {}}
          >
            <img
              alt="icon option"
              className="css-fbcrhc"
              src="test-file-stub"
            />
          </div>
        </a>
      </div>
      <div
        className="css-csay8j"
      >
        <a
          href="/YoutubeInterface"
          onClick={[Function]}
        >
          <div
            style={Object {}}
          >
            <img
              alt="icon option"
              className="css-fbcrhc"
              src="test-file-stub"
            />
          </div>
        </a>
      </div>
      <div
        className="css-csay8j"
      >
        <a
          href="/myFolders"
          onClick={[Function]}
        >
          <div
            style={Object {}}
          >
            <img
              alt="icon option"
              className="css-fbcrhc"
              src="test-file-stub"
            />
          </div>
        </a>
      </div>
    </div>
    <h1>
      Loading
    </h1>
  </div>
</div>
`;

exports[`mainApp snapshot Inside Folder renders 1`] = `
<div
  className="App"
>
  <div
    className="flex-container css-1dc4ncl"
  >
    <div
      className="css-15s9vjo"
    >
      <img
        alt="Frame Cards Logo"
        className="css-7r85ak"
        src="test-file-stub"
      />
    </div>
    <div
      className="css-ku0l26"
    >
      <form
        action="/someServlet"
        className="css-1rerb9r"
      >
        <button
          className="material-icons css-9h4d91"
          type="submit"
        >
          <i
            className="material-icons"
          >
            search
          </i>
        </button>
        <input
          className="css-pux1ie"
          name="search-input"
          placeholder="Search.."
          type="text"
        />
      </form>
    </div>
  </div>
  <div
    id="main"
  >
    <div
      className="flex-container css-8jr5nd"
      id="side"
    >
      <div
        className="css-csay8j"
      >
        <a
          href="/myFolders"
          onClick={[Function]}
        >
          <div
            style={Object {}}
          >
            <img
              alt="icon option"
              className="css-fbcrhc"
              src="test-file-stub"
            />
          </div>
        </a>
      </div>
      <div
        className="css-csay8j"
      >
        <a
          href="/CreateCard"
          onClick={[Function]}
        >
          <div
            style={Object {}}
          >
            <img
              alt="icon option"
              className="css-fbcrhc"
              src="test-file-stub"
            />
          </div>
        </a>
      </div>
      <div
        className="css-csay8j"
      >
        <a
          href="/CreateFolder"
          onClick={[Function]}
        >
          <div
            style={Object {}}
          >
            <img
              alt="icon option"
              className="css-fbcrhc"
              src="test-file-stub"
            />
          </div>
        </a>
      </div>
      <div
        className="css-csay8j"
      >
        <a
          href="/ImageInterface"
          onClick={[Function]}
        >
          <div
            style={Object {}}
          >
            <img
              alt="icon option"
              className="css-fbcrhc"
              src="test-file-stub"
            />
          </div>
        </a>
      </div>
      <div
        className="css-csay8j"
      >
        <a
          href="/YoutubeInterface"
          onClick={[Function]}
        >
          <div
            style={Object {}}
          >
            <img
              alt="icon option"
              className="css-fbcrhc"
              src="test-file-stub"
            />
          </div>
        </a>
      </div>
      <div
        className="css-csay8j"
      >
        <a
          href="/myFolders"
          onClick={[Function]}
        >
          <div
            style={Object {}}
          >
            <img
              alt="icon option"
              className="css-fbcrhc"
              src="test-file-stub"
            />
          </div>
        </a>
      </div>
    </div>
    <div
      className="css-1rpgtf9"
    >
      <h1>
        Your Folder has 0 cards
      </h1>
    </div>
  </div>
</div>
`;

exports[`mainApp snapshot My Folders renders 1`] = `
<div
  className="App"
>
  <div
    className="flex-container css-1dc4ncl"
  >
    <div
      className="css-15s9vjo"
    >
      <img
        alt="Frame Cards Logo"
        className="css-7r85ak"
        src="test-file-stub"
      />
    </div>
    <div
      className="css-ku0l26"
    >
      <form
        action="/someServlet"
        className="css-1rerb9r"
      >
        <button
          className="material-icons css-9h4d91"
          type="submit"
        >
          <i
            className="material-icons"
          >
            search
          </i>
        </button>
        <input
          className="css-pux1ie"
          name="search-input"
          placeholder="Search.."
          type="text"
        />
      </form>
    </div>
  </div>
  <div
    id="main"
  >
    <div
      className="flex-container css-8jr5nd"
      id="side"
    >
      <div
        className="css-csay8j"
      >
        <a
          href="/myFolders"
          onClick={[Function]}
        >
          <div
            style={Object {}}
          >
            <img
              alt="icon option"
              className="css-fbcrhc"
              src="test-file-stub"
            />
          </div>
        </a>
      </div>
      <div
        className="css-csay8j"
      >
        <a
          href="/CreateCard"
          onClick={[Function]}
        >
          <div
            style={Object {}}
          >
            <img
              alt="icon option"
              className="css-fbcrhc"
              src="test-file-stub"
            />
          </div>
        </a>
      </div>
      <div
        className="css-csay8j"
      >
        <a
          href="/CreateFolder"
          onClick={[Function]}
        >
          <div
            style={Object {}}
          >
            <img
              alt="icon option"
              className="css-fbcrhc"
              src="test-file-stub"
            />
          </div>
        </a>
      </div>
      <div
        className="css-csay8j"
      >
        <a
          href="/ImageInterface"
          onClick={[Function]}
        >
          <div
            style={Object {}}
          >
            <img
              alt="icon option"
              className="css-fbcrhc"
              src="test-file-stub"
            />
          </div>
        </a>
      </div>
      <div
        className="css-csay8j"
      >
        <a
          href="/YoutubeInterface"
          onClick={[Function]}
        >
          <div
            style={Object {}}
          >
            <img
              alt="icon option"
              className="css-fbcrhc"
              src="test-file-stub"
            />
          </div>
        </a>
      </div>
      <div
        className="css-csay8j"
      >
        <a
          href="/myFolders"
          onClick={[Function]}
        >
          <div
            style={Object {}}
          >
            <img
              alt="icon option"
              className="css-fbcrhc"
              src="test-file-stub"
            />
          </div>
        </a>
      </div>
    </div>
    <h1>
      loading
    </h1>
  </div>
</div>
`;

exports[`mainComponents snapshot SideBar renders 1`] = `
<div
  className="flex-container css-8jr5nd"
  id="side"
>
  <div
    className="css-csay8j"
  >
    <a
      href="/myFolders"
      onClick={[Function]}
    >
      <div
        style={Object {}}
      >
        <img
          alt="icon option"
          className="css-fbcrhc"
          src="test-file-stub"
        />
      </div>
    </a>
  </div>
  <div
    className="css-csay8j"
  >
    <a
      href="/CreateCard"
      onClick={[Function]}
    >
      <div
        style={Object {}}
      >
        <img
          alt="icon option"
          className="css-fbcrhc"
          src="test-file-stub"
        />
      </div>
    </a>
  </div>
  <div
    className="css-csay8j"
  >
    <a
      href="/CreateFolder"
      onClick={[Function]}
    >
      <div
        style={Object {}}
      >
        <img
          alt="icon option"
          className="css-fbcrhc"
          src="test-file-stub"
        />
      </div>
    </a>
  </div>
  <div
    className="css-csay8j"
  >
    <a
      href="/ImageInterface"
      onClick={[Function]}
    >
      <div
        style={Object {}}
      >
        <img
          alt="icon option"
          className="css-fbcrhc"
          src="test-file-stub"
        />
      </div>
    </a>
  </div>
  <div
    className="css-csay8j"
  >
    <a
      href="/YoutubeInterface"
      onClick={[Function]}
    >
      <div
        style={Object {}}
      >
        <img
          alt="icon option"
          className="css-fbcrhc"
          src="test-file-stub"
        />
      </div>
    </a>
  </div>
  <div
    className="css-csay8j"
  >
    <a
      href="/myFolders"
      onClick={[Function]}
    >
      <div
        style={Object {}}
      >
        <img
          alt="icon option"
          className="css-fbcrhc"
          src="test-file-stub"
        />
      </div>
    </a>
  </div>
</div>
`;

exports[`mainComponents snapshot header renders 1`] = `
<div
  className="flex-container css-1dc4ncl"
>
  <div
    className="css-15s9vjo"
  >
    <img
      alt="Frame Cards Logo"
      className="css-7r85ak"
      src="test-file-stub"
    />
  </div>
  <div
    className="css-ku0l26"
  >
    <form
      action="/someServlet"
      className="css-1rerb9r"
    >
      <button
        className="material-icons css-9h4d91"
        type="submit"
      >
        <i
          className="material-icons"
        >
          search
        </i>
      </button>
      <input
        className="css-pux1ie"
        name="search-input"
        placeholder="Search.."
        type="text"
      />
    </form>
  </div>
</div>
`;
